services/subscriber/service.go

```diff
@@ -183,20 +183,28 @@ func (s *Service) createSubscription(se subEntry, mode string, destinations []st
 	default:
 		return nil, fmt.Errorf("unknown balance mode %q", mode)
 	}
-	writers := make([]PointsWriter, len(destinations))
-	stats := make([]writerStats, len(writers))
-	for i, dest := range destinations {
+	writers := []PointsWriter{}
```

Comment:
Why have you changed these? Usually slices have been pre-allocated for a reason.